build.gradle.kts

```diff
@@ -247,21 +246,23 @@ project(":api") {
     }
 }
 
-// Ensure that the WPILibVersioningPlugin is setup by setting the release type, if releaseType wasn't
-// already specified on the command line
-if (!hasProperty("releaseType")) {
-    WPILibVersion {
-        releaseType = ReleaseType.DEV
-    }
-}
-
 /**
- * @return [edu.wpi.first.wpilib.versioning.WPILibVersioningPluginExtension.version] value or null
+ * @return publishVersion property if exists, otherwise
+ * [edu.wpi.first.wpilib.versioning.WPILibVersioningPluginExtension.version] value or fallback
  * if that value is the empty string.
  */
-fun getWPILibVersion(): String? = if (WPILibVersion.version != "") WPILibVersion.version else null
+fun getWPILibVersion(fallback: String = "v0.0.0"): String {
+    if (project.hasProperty("publishVersion")) {
+        val publishVersion: String by project
+        return publishVersion
+    } else if (WPILibVersion.version != "") {
+        return WPILibVersion.version
+    } else {
+        return fallback
+    }
+}
 
-task<Wrapper>("wrapper") {
+tasks.withType<Wrapper> {
```

Comment:
This should be `tasks.withType<Wrapper>().configureEach { ...` now

Reply:
Fixed